web/html/src/manager/maintenance/maintenance-windows-details.js

```diff
@@ -0,0 +1,178 @@
+/* eslint-disable */
+'use strict';
+
+const React = require("react");
+const {Button} = require("components/buttons");
+const {ModalButton} = require("components/dialog/ModalButton");
+const {DeleteDialog} = require("components/dialog/DeleteDialog");
+const { TopPanel } = require('components/panels/TopPanel');
+const {Messages} = require("components/messages");
+const {Form} = require("components/input/Form");
+const {Check} = require("components/input/Check");
+const { BootstrapPanel } = require('components/panels/BootstrapPanel');
+
+const scheduleTypeToString = (scheduleType) => {
+    switch (scheduleType) {
+        case "MULTI":
+            return "Multi";
+        case "SINGLE":
+            return "Single";
+    }
+    return null;
+}
+
+class MaintenanceWindowsDetails extends React.Component {
+
+    constructor(props) {
+        super(props);
+
+        this.state = {
+            type: props.type,
+            calendarData: this.props.data.calendarData,
+            messages: [],
+        };
+    }
+
+    setCheck = (model) => {
+        model.strategy === "" && (model.strategy = false);
+        this.setState(model);
+    };
+
+    addStrategy = () => {
+        const item = this.props.data;
+        item.strategy = (this.state.strategy ? "Cancel" : "Fail");
+        return item;
+    };
+
+    renderScheduleDetails(data) {
+        return (
+            <BootstrapPanel title={t("Schedule Details")}>
+                <div className="table-responsive">
+                    <table className="table">
+                        <tbody>
+                        <tr>
+                            <td>{t("Schedule Name")}</td>
+                            <td>{t(data.scheduleName)}</td>
```

Comment:
IIRC @mcalmer already caught me in this pattern: we cannot parametrize values inside the `t()` function because texts will be extracted and translated before runtime, do I remember correctly @mcalmer ? If so, we should receive a translated text value from the backend first.
Anyway, I am afraid I am a bit lost in this: is this a field value that the user populate so we should not translate at all or? It would be nice if you can add even some rough and wip screenshot in this PR to get a picture of what we are talking about. Thank you in advance :)

Reply:
Right this value is populated by the user, so it's not translated.